### PR TITLE
Added avatar css

### PR DESCRIPTION
**Pull request by tanishq20**
Please use the below link for a live preview
https://deploy-preview-1--compact-ui.netlify.app/docs/components/avatar.html

### CODE REVIEW

File: docs/stylesheets/base/utilities.css [hunk 1]
```diff
@@ -6,3 +12,11 @@
   gap: 20px;
   padding: 10rem;
 }
+
+.showcase-avatar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  margin-top: 2rem;
```
Comment (ttahm3d) [inline docs/stylesheets/base/utilities.css]: Try to use padding more times than margin. It is good practice as padding is always inside the content box in box-model

**Review by ttahm3d — COMMENTED** [review]
Great work. Didn't see any repeated CSS or any other stuff.

File: docs/stylesheets/base/utilities.css (hunk 1 above)
Comment (tanishq20) [reply docs/stylesheets/base/utilities.css]: Thanks, Tahir
Will change it

File: docs/stylesheets/base/utilities.css [hunk 2]
```diff
@@ -1,3 +1,9 @@
+.active-state {
+  color: var(--active-hover);
```
Comment (krishanu2911) [inline docs/stylesheets/base/utilities.css]: Good that you have used CSS variables

**Pull request merged**